back/boxtribute_server/mobile_distribution/crud.py

```diff
@@ -92,6 +95,88 @@ def change_distribution_event_state(distribution_event_id, distribution_event_st
     return distribution_event
 
 
+def set_products_for_packing_list(
+    user_id, distribution_event_id, product_ids_to_add, product_ids_to_remove
```

Comment:
Personally I like forcing the calling code to use keyword-argument form (`set_products(user_id=g.user.id, distribution_event_id=distribution_event_id`). Even though there is increased verbosity, the risk of misordering arguments is avoided.
Forcing keyword-arguments works by adding a `*` 'indicator' argument in the signature